distribution/docs/src/main/resources/content/_managing/_configuring/environment-hardening.adoc

```diff
@@ -102,4 +102,29 @@ a|Update the Session configuration to have no greater than a 10 minute Session T
  * Select the *Configuration* tab. +
  * Select `Session`. +
  * Set `Session Timeout (in minutes)` to `10` (or less). +
+
+|Shell command access
```

Comment:
✏️ `Shell Command Access` because the other values in this column are capitalized